lib/bigquery/index.js

```diff
@@ -96,15 +96,10 @@ var SCOPES = ['https://www.googleapis.com/auth/bigquery'];
  */
 function BigQuery(options) {
   if (!(this instanceof BigQuery)) {
+    options = util.normalizeArguments(this, options);
```

Comment:
I should have said move the normalize call below the if, so that the `if (!(this ...` is still the first thing to happen.